### ISSUE TITLE
Add Support for Avro 1.9.0

### ISSUE THREAD

**Issue by galvo** [confluentinc/common#210]
Would be good to see Confluent and the schema registry supporting the recent release of Avro 1.9.0 http://avro.apache.org/docs/1.9.0/, the first release in two years.

**Comment by leosilvadev** [comment]
Good point! I would be open to help to work on this. I am currently doing some workaround to use Avro 1.9 to build my Source connectors

**Comment by OneCricketeer** [comment]
Should probably close this https://github.com/rayokota/schema-registry/commit/c2c78ffc8f552746bc812bf3596f0cf2d701d143
